wiki/Rules/hu.md

```diff
@@ -0,0 +1,79 @@
+6.  **Legyél figyelmes, hogy mit osztasz meg másokkal, különös tekintettel a felkavaró/felzaklató tartalmakra vonatkozóan.** Ha azon tűnődsz, hogy helyénvaló-e az adott tartalmat linkelni, valószínűleg nem az. Kerüld a gyorsan villogó fényeket, a véres, másokat felkavaró, és egyéb felzaklató képeket.
+7.  **A linkek legyenek relevánsak és legális weboldalakra mutassanak.** Illegálisan közvetítő weboldalak (hivatalosan licenceltek, például Animelab vagy Crunchyroll rendben van) vagy bármiféle illegális/kalóz szoftverek, illetve ehhez hasonlóak nem elfogadottak, és nemkívánatosak a chat-en. Twitch linkek reklámozása más videojátékokra rendben van, de CSAK a `#videogames` csatornában.
+8. **Ne hirdess semmiféle külső chat szervert, Discord-ot, Teamspeak-et, illetve Skype szervereket/csoportokat beleértve.** Az osu! chat elsősorban az _osu!_-ról szól, és nem arról, hogy a saját szervereidet híreszteld. Tedd ezt privát üzenetekben, ha mindenképpen szükséges.
+9.  **Nyelvspecifikus csatornákban, a domináns nyelvet kell beszélned (Angol az #english-ben, Magyar a #hungarian-ben, stb.) ha beszélgetésben veszel részt.** Abban a csatornában, ahol a nyelv nincs kifejezetten előírva (vagy a csatorna nevéből nem lehet kikövetkeztetni, például Magyar a `#hungarian`-ben és Holland a `#dutch`-ban), ott mindig Angol. A `#lobby` és a `#help` csatornákban bármely nyelv megengedett, nem úgy, mint az összes többiben. *A [moderátori csapatnak](/wiki/People/Global_Moderation_Team) nem kell követnie ezt a szabályt, amikor a munkájukat végzik.*
+
+Fórum Szabályzat
+----------------
+
+1.  **Csak értelmes, érdekes tartalmakat/témákat posztolj.** A fórum elsősorban megvitatásokra van, szóval győződj meg arról, hogy elég energiát szántál arra, hogy ne rabold a többiek idejét.
+2.  **Tartsd a posztjaidat és a válaszaidat a korhatár nélküli közösséghez megfelelően.** Felnőtt tartalomnak semmi keresnivalója egy korhatár nélküli közösségben, ugyanakkor semmi sem állít meg ezeknek a megvitatása privát üzenetekben.
+3.  **Legyél produktív a kritikáddal, anélkül, hogy átfolyna személyes támadásba.** A kritika egy nagyszerű dolog, ha rendesen van kivitelezve, de ha csupán személyes támadást eredményezel a kifejtéseddel, akkor rosszul csinálod, és el kéne gondolkozz ezen.
+4.  **Tartsd a szálakat relevánsan, és ne élessz fel régi témákat, ha kihaltak a beszélgetések.** Ha a téma már évek óta halott, nem igazán releváns többé, hagyd azt eltűnni. Posztolj újat, vagy keress és nézd meg, hogy valaki nem-e válaszolta meg már a kérdésed előtte.
+5.  **Az avatarjaid és az aláírásaid maradjanak ésszerűek.** Ha animáltak, ne villogjanak túlságosan. Egyúttal, a fájlméretük legyen alacsony, hogy a lassabb internetcsatlakozással rendelkezők is kényelmesen tudják böngészni az oldalt. Ez ideálisan maximum fél megabyte (500 kilobyte), ahol lehetséges.
```

Comment:
"Ezen kívül" instead of "Egyúttal"? That word sounds weird at the start of a new sentence.